sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/NCover3ReportParser.java

```diff
@@ -56,6 +57,8 @@ public void accept(File file, Coverage coverage) {
     }
 
     public void parse() {
+      LOG.warn("NCover3 coverage import is deprecated since version 8.6 of the plugin. " +
```

Comment:
An idea would be to display this also on SQ/SC (using AnalysisWarnings) to increase visibility.